consul/service.sls

```diff
@@ -0,0 +1,12 @@
+include:
+  - .install
+  - .config
```

Comment:
Service shouldn't depend directly on any other states. It should simply handle starting/restarting the service so that it can be included in other states without automatically running the other states.